flipperdevices/flipperzero-firmware
# [FL-2931] Storage full RPC error handling

# What's new

- Incomplete writes due to the storage being full are now treated as errors.

# Verification 

1. Compile and upload.
2. Fill the SD card almost completely (leave a couple of megs free).
3. Use an RPC client to write a file larger than the free space.

Flipper should return a `PB_CommandStatus_ERROR_STORAGE_INTERNAL` when the space runs out.

# Caveats
- QFlipper does not handle this error (yet), a fix is in the works.

# Checklist (For Reviewer)
